src/@next/components/atoms/SocialMediaIcon/stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
```

Comment:
maybe icons should be in subfolder? Something like `/atoms/icons/SocialMediaIcon`.

Reply:
I don't think there will be more than this one so I'm not sure if they need to be put inside an additional subtree. I thought about reimplementing the Icon component itself, as in: use paths instead of SVG. Currently the SVGs all have mixed/non-standarized viewport sizes (some are not even squares), by using a path we can create an Icon component that renders an SVG inside 32x32 (the most popular) viewport by default and by using a "size" prop we can specify the exact size, meaning having 2 different icons next to each other would keep the same size. We would also not need to provide/import an SVG, but rather use a union of type `string` to specify what Icon we want. Here's a use case:
```
<Icon name="trash" size={24} />
<Icon name="social_facebook" size={40} />
```

There's one downside to this I can think of: We would need to transform these SVGs to paths. Most of them will be easy to do, but some will require Adobre Ilustrator to get rid of strokes. What do you think? (cc: @AlicjaSzu  @mateuszkula)

Reply:
This change sounds reasonable - I see a place for improvement in my code if we would use such approach :) Fine by me

Reply:
I'll open new PR with this change